RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.cc

```diff
@@ -315,65 +310,61 @@ void PCCNTupler::analyze(const edm::Event& iEvent,
         }
     }
 
-
-    // -- Does this belong into beginJob()?
-    ESHandle<TrackerGeometry> TG;
-    iSetup.get<TrackerDigiGeometryRecord>().get(TG);
-
+    int NumPixelBarrelLayers=3;
+    if(pixelPhase2Geometry){
+      NumPixelBarrelLayers=4;
```

Comment:
I'm curious why is this only for phase2?
Doesn't phase-1 have the same feature?